app/views/docs/databases.phtml

```diff
@@ -33,23 +33,18 @@
 
 <h3><a href="/docs/databases#permissions" id="permissions">Permissions</a></h3>
 <p>
-    Once you create your collection, you will reach your collection's settings page, where you can choose the permission model for your collection. There are two types of permissions models available, each one with different pros and cons designed to be flexible to best match your use case.
+    Appwrite provides permissions to restrict access to documents at two levels, document and collection level. When a user has the appropriate type of <a href="/docs/permissions/">access permissions</a> granted at **either** the document or the collection level, it will be able to view or make changes to the document. If the permission field is left empty, no one can access the document.
 </p>
-
 <h4>Document Level Permissions</h4>
-<p>
-    With this permission model, you have granular access control over every document, and users will only be able to access documents for which they have explicit permissions. Document permissions are required in this permission model, and collection permissions are optional.
-</p>
+<p>Document-level permissions grant access to individual documents. Document level permissions are only enabled if Document Security is enabled in the settings of your collection.</p>
```

Comment:
Should we be more consistent with Document Level vs Document-Level?

Reply:
It should never be document-level